pkgs/tools/compression/xz/xz-after-bootstrap.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  automake,
+  autoconf,
+  perl,
+  libtool,
+  gettext,
+  perl538Packages,
+  xz,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xz";
+  version = xz.version;
+
+  src = fetchFromGitHub {
+    owner = "tukaani-project";
+    repo = "xz";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Xp1uLtQIoOG/qVLpq5D/KFmTOJ0+mNkNclyuJsvlUbE=";
+  };
+
+  strictDeps = true;
+
+  outputs = [
+    "bin"
+    "dev"
+    "out"
+    "man"
+    "doc"
+  ];
+
+  enableParallelBuilding = true;
+
+  doCheck = true;
+
+  nativeBuildInputs = [
+    gettext
+    libtool
+    automake
+    autoconf
```

Comment:
Maybe in general it'd be better to use `.overrideAttrs`, so the inputs will stay the same, up to `auto*` tools of course.